boards/sparkfun/micromod/micromod_nrf52840_defconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable I2C
+CONFIG_I2C=y
+
+# enable SPI
+CONFIG_SPI=y
```

Comment:
Why is i2c and spi enabled by default?

Reply:
I have removed them, I forgot to eliminate from here.